src/io/flutter/actions/InstallSdkAction.java

```diff
@@ -38,9 +38,7 @@
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-@SuppressWarnings("ComponentNotRegistered")
```

Comment:
Please leave this annotation. Removing it causes a warning in my dev env (Android Studio).